sdk/imagebuilder/azure-resourcemanager-imagebuilder/src/test/java/com/azure/resourcemanager/imagebuilder/ImageBuilderManagerTests.java

```diff
@@ -0,0 +1,145 @@
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateImageTemplate() {
+        ImageTemplate imageTemplate = null;
+        try {
+            String randomPadding = randomPadding();
+            String templateName = "template" + randomPadding;
+            String imageName = "image" + randomPadding;
+            String identityName = "identity" + randomPadding;
+            String imageId = resourceManager.resourceGroups().getByName(resourceGroupName).id() + "/providers/Microsoft.Compute/images/" + imageName;
+
+            Identity identity = msiManager.identities()
+                .define(identityName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withAccessToCurrentResourceGroup(BuiltInRole.CONTRIBUTOR)
+                .create();
+            Map<String, UserAssignedIdentity> userAssignedIdentities = new HashMap<>();
+            userAssignedIdentities.put(identity.id(), new UserAssignedIdentity());
+
+            imageTemplate = imageBuilderManager.virtualMachineImageTemplates()
+                .define(templateName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withIdentity(
+                    new ImageTemplateIdentity()
+                        .withType(ResourceIdentityType.USER_ASSIGNED)
+                        .withUserAssignedIdentities(userAssignedIdentities))
+                .withDistribute(Arrays.asList(
+                    new ImageTemplateManagedImageDistributor()
+                        .withImageId(imageId)
+                        .withLocation(REGION.name())
+                        .withRunOutputName("runOutputManagedImage")
+                    )
+                )
+                .withVmProfile(new ImageTemplateVmProfile().withVmSize("Standard_DS1_v2").withOsDiskSizeGB(32))
+                .withSource(
+                    new ImageTemplatePlatformImageSource()
+                        .withPublisher("canonical")
+                        .withOffer("0001-com-ubuntu-server-focal")
+                        .withSku("20_04-lts-gen2")
+                        .withVersion("latest"))
+                .withBuildTimeoutInMinutes(0)
+                .create();
+            // @embedEnd
+            imageTemplate.refresh();
+            Assertions.assertEquals(imageTemplate.name(), templateName);
+            Assertions.assertEquals(imageTemplate.name(), imageBuilderManager.virtualMachineImageTemplates().getById(imageTemplate.id()).name());
+            Assertions.assertTrue(imageBuilderManager.virtualMachineImageTemplates().listByResourceGroup(resourceGroupName).stream().findAny().isPresent());
+        } finally {
+            if (imageTemplate != null) {
+                imageBuilderManager.virtualMachineImageTemplates().deleteById(imageTemplate.id());
+            }
```

Comment:
Delete the identity after this.

Does it require a user-assigned identity? Can system-assigned identity work?

Reply:
1. Yes, user-assigned identity is required to use imagebuilder, system-assigned identity is not supported.
2. `Delete the identity after this.` has been fixed in the new version.